packages/ui/src/input/index.css

```diff
@@ -1,43 +1,42 @@
-input {
+input.osc {
```

Comment:
Ook hier misschien `.osc input`  gebruiken? Daardoor hoef je geen class meer mee te geven aan de input? Geen idee wat dat in andere omgevingen doet eigenlijk

Reply:
Aangepast